### ISSUE TITLE
click latest release as a dependency to typer causes error

### ISSUE THREAD

**Issue by stezskal** [pallets/click#2230]
Hi - I use typer and it has click as a dependency.  The latest click release 8.1.0 does not work and gives the error below - 8.0.4 works so my workaround is to explicitly specify 8.0.4.  

```
Traceback (most recent call last):
  File  line 158, in <module>
    import typer
  File "/lib/python3.7/site-packages/typer/__init__.py", line 12, in <module>
    from click.termui import get_terminal_size as get_terminal_size
    ImportError: cannot import name 'get_terminal_size' from 'click.termui' (/lib/python3.7/site-packages/click/termui.py)
```

I know it's not a lot of information but maybe it means something to developers here.  Not sure if this is more appropriate to post as a typer issue.

**Comment by stezskal** [comment]
This is already addressed in the typer repo.

**Comment by davidism** [comment]
This was already issuing a deprecation warning, documented as such in the changelog, and then documented as removed in the changelog.

Use a tool like [pip-tools](https://pypi.org/project/pip-tools/) to pin your dependency tree and control when you get updates. Be sure to run your tests with deprecation warnings treated as errors so that you get notified of these types of changes early.

**Comment by davidism** [comment]
The typer maintainer is currently at a conference, I'm sure they'll make a release when they get back and have the ability to.

**Comment by tiangolo** [comment]
Hello! I just released Typer `0.4.1` that should handle this. :rocket: :nerd_face:

Thanks for the conversation and for understanding!

**Comment by pombredanne** [comment]
FWIW MO is that this and https://github.com/pallets/click/issues/2232 and https://github.com/psf/black/issues/2964  and https://github.com/pallets/click/issues/2228caused roughly 1/2 of the world's Python CI jobs to break ;)

> Use a tool like pip-tools to pin your dependency tree and control when you get updates. Be sure to run your tests with deprecation warnings treated as errors so that you get notified of these types of changes early.


This is usually not an option as various libraries would pin on different versions and this will make things fail even more. 

May I suggest  this should have been a version 9 rather than a version 8.1?
